### PR TITLE
Test

### PR DESCRIPTION
**Pull request by silverdaz**
To see if I keep the comments after I delete the branch

### CODE REVIEW

**Comment by silverdaz** [comment]
That's a great comment

**Comment by silverdaz** [comment]
That's another one

**Comment by silverdaz** [comment]
Pjew... I'm on fire

File: src/lega/conf/defaults.ini [hunk 1]
```diff
@@ -78,16 +78,6 @@ interval = 60
 home = /ega/inbox/%(user_id)s
 home_skel = /ega/skel
 
-# [master.key.1]
-# key = /path/to/rsa/key.pem
-# pub_key = /path/to/rsa/public-key.pem
-# passphrase = something
-
```
Comment (viklund) [inline src/lega/conf/defaults.ini]: Another code comment.

**Pull request closed without merging**